ocs_ci/framework/pytest_customization/reports.py

```diff
@@ -114,8 +114,10 @@ def pytest_sessionfinish(session, exitstatus):
                 ]
                 c.writerow(row)
         logger.info(f"Test Time report saved to '{time_report_file}'")
-    except Exception:
-        logger.exception("Failed to save Test Time report to logs directory")
+    except (Exception, FileNotFoundError) as e:
```

Comment:
The `FileNotFoundError` a child of the general `Exception`, so it is not necessary to mention it here:
```suggestion
    except Exception as e:
```

The traceback is printed to the logs because it is automatically added by the [`logger.exception(...)`](https://docs.python.org/3/library/logging.html#logging.Logger.exception) method. You probably want to change the log level to _debug_, _info_ or _warning_ (not sure if it is necessary to print it with `error` level).